Stanford/JumpstartLab/Install/ContextSettings.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Stanford\JumpstartLab\Install;
+
+use \ITasks\AbstractInstallTask;
+
+/**
+ * Class ContextSettings.
+ *
+ * @package Stanford\JumpstartLab\Install
+ */
+class ContextSettings extends AbstractInstallTask {
+
+  /**
+   * Disable some contexts.
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
```

Comment:
I'm unsure if they might use it. think I'll delete that one from jsl_layouts and keep the one for people layouts